### PR TITLE
New pr [pullrequest]

### PR DESCRIPTION
**Pull request by tylermann**
test pr

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -4,14 +4,23 @@ def func():
     pass
 
 
+def one_up_here():
+    pass
```
Comment (tylermann) [inline test.py:11]: is this doing anything?

**Comment by tylermann** [comment]
test

**Comment by tylermann** [comment]
test2

**Comment by tylermann** [comment]
test3

File: test.py (hunk 1 above)
Comment (tylermann) [reply test.py:11]: doesn't look like it

**Comment by tylermann** [comment]
test4

**Comment by tylermann** [comment]
test5

**Comment by tylermann** [comment]
test6

**Comment by tylermann** [comment]
test7

**Comment by tylermann** [comment]
test8

**Comment by tylermann** [comment]
test9

**Comment by tylermann** [comment]
test10

**Comment by tylermann** [comment]
test11

**Comment by tylermann** [comment]
test12

**Review by pbarsallo — CHANGES_REQUESTED** [review]
need some changes here

**Comment by tylermann** [comment]
test13

**Review by ghost — COMMENTED** [review]
:thumbsup:

**Review by ghost — COMMENTED** [review]
:thumbsdown:

**Review by ghost — COMMENTED** [review]
test

File: test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tylermann) [reply test.py:11]: okay

[the remark this answers is not in the document]
Comment (tylermann) [reply test.py:11]: done

File: test.py [hunk 2]
```diff
@@ -7,14 +7,24 @@ def func():
...
+
+
 def new_function():
     a = 1
     a += 1
     a += 2
```
Comment (tylermann) [inline test.py:17]: two

File: test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (danMateer) [reply test.py:11]: Reply 01182018

**Comment by danMateer** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/tylermann/public_test/16)*

**Pull request closed without merging**